connect_cmd.go

```diff
@@ -98,6 +99,10 @@ func beforeConnectAction(ctx *cli.Context) error {
 		return err
 	}
 
+	if ContentFeature.Enabled == false && len(contentTemplates) > 0 {
```

Comment:
Looks like there is a lint suggestion here that should be resolved.

Reply:
Yeah, I know, I will fix it, but to be honest, I do not like it ... it is so easy to overlook it ... See: `!Item`